aws/v16.3.1/release.yaml

```diff
@@ -66,7 +66,7 @@ spec:
   - name: kubernetes
     version: 1.21.8
   date: "2022-01-19T10:23:42Z"
-  state: active
+  state: deprecated
```

Comment:
Since this is a new minor, we normally don't do this.
Unless we're fine making customers skip minors when they upgrade

Reply:
I was under impression we only leave one active release per major

Reply:
but dont have any strong feelings about that

Reply:
on azure we do that for minor (we don't allow skipping minors).
but if it works on AWS I'm ok

Reply:
@alex-dabija WDYT?

Reply:
On AWS we skip minors and we deprecate the previous release.